CONTRIBUTING.md

```diff
@@ -1,15 +1,45 @@
-## How to contribute to LINE Bot SDK for Ruby project
+# How to contribute to LINE Bot SDK for Ruby project
 
-First of all, thank you so much for taking your time to contribute! LINE Bot SDK for Ruby is not very different from any other open
-source projects you are aware of. It will be amazing if you could help us by doing any of the following:
+First of all, thank you so much for taking your time to contribute!
+LINE Bot SDK for Ruby is not very different from any other open source projects you are aware of.
+It will be amazing if you could help us by doing any of the following:
 
-- File an issue in [the issue tracker](https://github.com/line/line-bot-sdk-ruby/issues) to report bugs and propose new features and
-  improvements.
+- File an issue in [the issue tracker](https://github.com/line/line-bot-sdk-ruby/issues) to report bugs and propose new features and improvements.
 - Ask a question using [the issue tracker](https://github.com/line/line-bot-sdk-ruby/issues).
 - Contribute your work by sending [a pull request](https://github.com/line/line-bot-sdk-ruby/pulls).
 
+## Development
```

Comment:
(We should also address the OpenAPI code generator, but for this PR, it is considered out of scope.)